### PR TITLE
Sg 1524 dept refactor

### PR DESCRIPTION
**Pull request by aekong**
* remove a bunch of unnecessary preprocessor code and update templates accordingly
* bring in the parts of the sfgov design system that makes sense (for now)

### CODE REVIEW

File: web/themes/custom/sfgovpl/templates/field/department_full/field--node--field-social-media--department.html.twig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% if items is not empty %}
+  <p class="title">{{ label }}</p>
+  <ul class="sfgov-social-list">
+    {% for item in items %}
+      {% set facebook = item.content['#paragraph'].get('field_facebook').getValue()[0].uri %}
+      {% set instagram = item.content['#paragraph'].get('field_instagram').getValue()[0].uri %}
+      {% set twitter = item.content['#paragraph'].get('field_twitter').getValue()[0].uri %}
```
Comment (arlina-espinoza) [inline web/themes/custom/sfgovpl/templates/field/department_full/field--node--field-social-media--department.html.twig:7]: If an item is empty, does it throw a warning for undefined indexes or not?

**Review by arlina-espinoza — APPROVED** [review]
Hi @aekong. This looks really good to me and I'm happy to see how much more ordered the codebase looks, but I'm a little bit nervous about the major refactor to this section. Do you think someone else could QA several use cases (empty fields, etc) before merging?

**Comment by aekong** [comment]
Yup totally, asked our design team to review as well! Thanks!

File: web/themes/custom/sfgovpl/templates/field/department_full/field--node--field-social-media--department.html.twig (hunk 1 above)
Comment (aekong) [reply web/themes/custom/sfgovpl/templates/field/department_full/field--node--field-social-media--department.html.twig:7]: looks like no warning if the social media paragraph is empty 😅

**Pull request merged**